public/js/routes/station-detail/station-detail-view.js

```diff
@@ -109,6 +113,17 @@ var StationDetailView = View.extend({
    */
   _triggerPredictionsChange: function() {
     this.trigger( 'change:trips' );
+    window.tbd = this.tripsByDirection;
+  },
+
+  render: function() {
+    this.renderWithTemplate( this );
+
+    // Get a reference to the container for the lists of predictions by direction
+    this.predictionsLists = this.queryByHook( 'predictions-lists' );
+
+    // Render out a view for each direction
+    this.renderCollection( this.tripsByDirection, PredictionListView, this.predictionsLists );
```

Comment:
It is possible that the extra overhead of `renderCollection` (i.e., the need for `tripsByDirection` to itself be a fully-formed collection, with its own model) may be overkill for this purpose